packages/forma-36-react-components/src/components/Autocomplete/README.mdx

```diff
@@ -0,0 +1,37 @@
+# Autocomplete (alpha)
+
+The autocomplete component is an input field that provides selectable suggestions as a user types into it. It allows users to quickly search through and select from large collections of options.
+Keep in mind that this component is still in alpha state, which means it's unsupported and subject to breaking changes without warning.
+
+## Example of usage
+
+```jsx
+import {Autocomplete } from '@contentful/forma-36-react-components/dist/alpha';
+
+<Autocomplete
+  items={items}
+  width={'full'}
+  onQueryChange={onQueryChange}
+  onChange={onChange}
+  isLoading={isLoading}
+  placeholder={'search'}
+  emptyListMessage={'no result found'}
+  noMatchesMessage={'no matches'}
+  dropdownProps={{ isFullWidth: true }}
+  disabled={disabled}
+  {(options) => options.map((option) => <span key={option.value}>{option.label}</span>)}
+</Autocomplete>
+```
+
+## Content recommendations
+
+ - Autocomplete label should be short, contain 1 to 3 words
+ - Label should be written in a sentence case (the first word capitalized, the rest lowercase)
```

Comment:
@restiles maybe you would have suggestions when it comes to Autocomplete? Maybe some placeholder recommendation? 🙇‍♀️

Reply:
My recomendation is something that I'm always discussing in my feature team hahahah

If you have a list with 1000 items, DO NOT use select boxes. Use the autocomplete
Select box for 10 options, more than that use autocomplete pls 🙏 

at Contentful we have a select box for selecting a country. It's a list with all countries, all 193 of them
If you live in Zimbabwe...